pkgs/development/interpreters/clojure/babashka.nix

```
@@ -1,24 +1,25 @@
-{ stdenv, fetchurl, graalvm8 }:
+{ stdenv, fetchurl, graalvm8, glibcLocales }:
 
 with stdenv.lib;
 stdenv.mkDerivation rec {
   pname = "babashka";
-  version = "0.0.78";
+  version = "0.0.88-2";
 
   reflectionJson = fetchurl {
     name = "reflection.json";
     url = "https://github.com/borkdude/${pname}/releases/download/v${version}/${pname}-${version}-reflection.json";
-    sha256 = "1m1nwdxjsc6bkdzkbsll316ly0c3qxaimjzyfph1220irjxnm7xf";
+    sha256 = "08aggymd8s43qq8y73aw33dbnvfypmg662rfzlqzjs7b4f8d35nj";
   };
 
   src = fetchurl {
     url = "https://github.com/borkdude/${pname}/releases/download/v${version}/${pname}-${version}-standalone.jar";
-    sha256 = "01w990zk5qjrbnc846snh6na002kdyrlrfnqwg03ibx20g3mr7if";
+    sha256 = "1hv37lvpvyxq7788kd3ig90mwzr72h3za30yhk9gcwynmnga0gin";
   };
 
   dontUnpack = true;
 
-  buildInputs = [ graalvm8 ];
+  LC_ALL = "en_US.UTF-8";
+  buildInputs = [ graalvm8 glibcLocales ];
```

`glibcLocales` is used at build time. Hence it should be put into `nativeBuildInputs`.

---

Same applies to `graalvm8`:

```suggestion
  nativeBuildInputs = [ graalvm8 glibcLocales ];
```

---

Corrected, thanks!